src/components/feedback/Popover.tsx

```diff
@@ -25,27 +25,43 @@ type PopoverCSSProps =
   | 'bottom'
   | 'marginTop';
 
+type PopoverPositioningRefs = {
+  anchorRef: RefObject<HTMLElement | undefined>;
+  popoverRef: RefObject<HTMLElement | undefined>;
+};
+
+type PopoverPositioningOptions = {
+  popoverOpen: boolean;
+  placement: 'above' | 'below';
+
+  /**
+   * Whether the popover should be aligned to the right side of the anchor
+   * element or not
+   */
+  alignToRight: boolean;
+
+  /** Native popover API is used to toggle the popover */
+  asNativePopover: boolean;
+};
+
 /**
  * Manages the popover position manually to make sure it renders "next" to the
  * anchor element (above or below). This is mainly needed when using the
  * popover API, as that makes it render in the top layer, making it impossible
  * to position it relative to the anchor element via regular CSS.
- *
- * @param asNativePopover - Native popover API is used to toggle the popover
- * @param alignToRight - Whether the popover should be aligned to the right side
- *                       of the anchor element or not
  */
 function usePopoverPositioning(
-  anchorElementRef: RefObject<HTMLElement | undefined>,
-  popoverRef: RefObject<HTMLElement | undefined>,
-  popoverOpen: boolean,
-  asNativePopover: boolean,
-  alignToRight: boolean,
-  placement: 'above' | 'below',
+  { anchorRef, popoverRef }: PopoverPositioningRefs,
```

Comment:
Since there are only two refs and both are required, it might be slightly nicer to pass them as positional arguments, rather than two separate options-like objects.